CHANGELOG.md

```diff
@@ -44,6 +44,10 @@ The types of changes are:
 * Fixed issue with extending the configuration [#721](https://github.com/ethyca/fidesops/pull/721)
 * Fix no such container error with docker-compose [758](https://github.com/ethyca/fidesops/pull/758)
 
+### Added
+* Adds SaaS connection type to SaaS yaml config [748](https://github.com/ethyca/fidesops/pull/748)
+
+
```

Comment:
We released 1.6.1 recently right? I'd get this up-to-date with main again and move this under "Unreleased"

Reply:
Ah yes, I'll update